pioinstaller/penv.py

```
@@ -69,67 +176,65 @@ def create_core_penv(penv_dir=None, ignore_pythons=None):
+    env["VIRTUAL_ENV"] = penv_dir
+
+    cmd = [system_uv_exe, "pip", "install", "uv"]
+
+    try:
+        subprocess.check_call(
+            cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=env
+        )
+        log.debug("Successfully installed uv in venv")
+    except subprocess.CalledProcessError as e:
+        log.debug("Failed to install uv in venv: %s", str(e))
+        raise exception.PIOInstallerException("Could not install uv in penv")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)